src/custom/constants/tokens/index.ts

```diff
@@ -49,6 +52,42 @@ export const V_COW: Record<number, Token> = {
   [SupportedChainId.RINKEBY]: V_COW_TOKEN_RINKEBY,
 }
 
+/**
+ * Cow token
+ */
+const COW_TOKEN_MAINNET = new Token(
+  SupportedChainId.MAINNET,
+  COW_CONTRACT_ADDRESS[SupportedChainId.MAINNET] || '',
+  18,
+  'COW',
+  'CoW Protocol Token'
+)
+
+const COW_TOKEN_XDAI = new Token(
+  SupportedChainId.XDAI,
+  COW_CONTRACT_ADDRESS[SupportedChainId.XDAI] || '',
+  18,
+  'COW',
+  'CoW Protocol Token'
+)
+
+const COW_TOKEN_RINKEBY = new Token(
+  SupportedChainId.RINKEBY,
+  COW_CONTRACT_ADDRESS[SupportedChainId.RINKEBY] || '',
+  18,
+  'COW',
+  'CoW Protocol Token'
+)
+
+export const COW: Record<number, Token> = {
```

Comment:
why `number`? it should be `SupportedChainId` using ours

Reply:
Yes the chainId is a number and we used this `V_COW[chainId]` already in few places.